tests/test_dataset.py

```diff
@@ -48,7 +48,8 @@ def test_shape(self, expected_shape):
                 'extra': os.path.join(tempdir, 'test_extra2.nii.gz')
             }
         ]
-        dataset = Dataset(data=test_data, transform=LoadNiftid(keys=['image', 'label', 'extra']))
+        test_transform = Compose([LoadNiftid(keys=['image', 'label', 'extra']), SimulateDelay(delay_time=1e-7)])
```

Comment:
Maybe we should use the dictionary type transform `SimulateDelayd` in this test, same as LoadNiftid?